lib/ansible/modules/storage/ibm/ibm_sa_host_ports.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (C) 2018 IBM CORPORATION
+# Author(s): Tzur Eliyahu <tzure@il.ibm.com>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'metadata_version': '1.1'}
+
+DOCUMENTATION = '''
+---
+module: ibm_sa_host_ports
+short_description: Add host ports on an IBM Spectrum Accelerate storage array.
+version_added: "2.8"
+
+description:
+    - "This module adds ports to or removes them from the hosts
+        on IBM Spectrum Accelerate storage systems."
+
+options:
+    host:
+        description:
+            - Host name.
+        required: true
+    state:
+        description:
+            - Host ports state.
+        required: true
+        default: "present"
+        choices: [ "present", "absent" ]
+    iscsi_name:
+        description:
+            - iSCSI initiator name.
+        required: false
+    fcaddress:
+        description:
+            - Fiber channel address.
+        required: false
+    num_of_visible_targets:
+        description:
+            - Number of visible targets.
+        required: false
+
+extends_documentation_fragment:
+    - ibm_storage
+
+author:
+    - Tzur Eliyahu (tzure@il.ibm.com)
```

Comment:
@tzurE In a follow up PR could you please update all your modules to be in the form

`- Tzur Eliyahu (@tzurE tzurE)`

This allows Ansibulbot to know who to contact if their are follow up issues or PRs

Reply:
I will.
Can I do this in one PR and add some documentation changes as well?

Reply:
@tzurE Single PR for all modules added in Ansible 2.8 would be great, and hopefully easiest for you as well.

Reply:
You can see how the module docs look at https://docs.ansible.com/ansible/devel/modules/list_of_storage_modules.html

This is the state of the `devel` branch, it's rebuild daily